### PR TITLE
static pages nad test

### PR DESCRIPTION
**Pull request by inuyashaaa**
------------------------------------------
Running Test
------------------------------------------

[+] Running:  rubocop --require rubocop/formatter/checkstyle_formatter --format RuboCop::Formatter::CheckstyleFormatter --no-color --rails --out .framgia-ci-reports/rubocop.xml app/ lib/
[+] Running:  brakeman -o .framgia-ci-reports/brakeman.html -o .framgia-ci-reports/brakeman.json
Loading scanner...
Processing application in /home/illuminati/Desktop/Trainning/project-2
Processing gems...
[Notice] Detected Rails 5 application
Processing configuration...
[Notice] Escaping HTML by default
Parsing files...
Processing initializers...
Processing libs...ed
Processing routes...          
Processing templates...       
Processing data flow in templates...
Processing models...          
Processing controllers...     
Processing data flow in controllers...
Indexing call sites...        
Running checks in parallel...
 - CheckBasicAuth
 - CheckBasicAuthTimingAttack
 - CheckCrossSiteScripting
 - CheckContentTag
 - CheckCreateWith
 - CheckDefaultRoutes
 - CheckDeserialize
 - CheckDetailedExceptions
 - CheckDigestDoS
 - CheckDynamicFinders
 - CheckEscapeFunction
 - CheckEvaluation
 - CheckExecute
 - CheckFileAccess
 - CheckFileDisclosure
 - CheckFilterSkipping
 - CheckForgerySetting
 - CheckHeaderDoS
 - CheckI18nXSS
 - CheckJRubyXML
 - CheckJSONEncoding
 - CheckJSONParsing
 - CheckLinkTo
 - CheckLinkToHref
 - CheckMailTo
 - CheckMassAssignment
 - CheckMimeTypeDoS
 - CheckModelAttrAccessible
 - CheckModelAttributes
 - CheckModelSerialize
 - CheckNestedAttributes
 - CheckNestedAttributesBypass
 - CheckNumberToCurrency
 - CheckQuoteTableName
 - CheckRedirect
 - CheckRegexDoS
 - CheckRender
 - CheckRenderDoS
 - CheckRenderInline
 - CheckResponseSplitting
 - CheckRouteDoS
 - CheckSafeBufferManipulation
 - CheckSanitizeMethods
 - CheckSelectTag
 - CheckSelectVulnerability
 - CheckSend
 - CheckSendFile
 - CheckSessionManipulation
 - CheckSessionSettings
 - CheckSimpleFormat
 - CheckSingleQuotes
 - CheckSkipBeforeFilter
 - CheckSQL
 - CheckSQLCVEs
 - CheckSSLVerify
 - CheckStripTags
 - CheckSymbolDoSCVE
 - CheckTranslateBug
 - CheckUnsafeReflection
 - CheckValidationRegex
 - CheckWithoutProtection
 - CheckXMLDoS
 - CheckYAMLParsing
Checks finished, collecting results...
Generating report...
Report saved in '.framgia-ci-reports/brakeman.html'
Report saved in '.framgia-ci-reports/brakeman.json'
[+] Running:  bundle-audit check --update > .framgia-ci-reports/bundle-audit.txt
From https://github.com/rubysec/ruby-advisory-db
 * branch            master     -> FETCH_HEAD
[+] Running:  rails_best_practices -e "db/schema.rb,db/migrate,vendor,app/models/application_record.rb" --format html --output-file .framgia-ci-reports/rails_best_practices.html app/
Source Code: |===================================================================================================================|
[+] Running:  eslint --format=checkstyle --output-file=.framgia-ci-reports/eslint.xml app/
[+] Running:  rspec --format html --out .framgia-ci-reports/rspec.html spec/
[+] Running:  scss-lint --require=scss_lint_reporter_checkstyle --format=Checkstyle --out=.framgia-ci-reports/scss-lint.xml app/assets/stylesheets/
[+] Running:  reek --format html > .framgia-ci-reports/reek.html

------------------------------------------
[o] Build Success!
------------------------------------------

[o] rubocop: success
[o] brakeman: success
[o] bundle-audit: success
[o] rails_best_practices: success
[o] eslint: success
[o] rspec: success
[o] scss-lint: success
[o] reek: success

### CODE REVIEW

**Comment by inuyashaaa** [comment]
ready

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -2,3 +2,5 @@
  *= require_tree .
  *= require_self
  */
+@import "bootstrap-sprockets";
```
Comment (dieunb) [inline app/assets/stylesheets/application.scss]: css nháy đơn, kiểm tra scss lint đi nhé

File: app/controllers/users/confirmations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
```
Comment (dieunb) [inline app/controllers/users/confirmations_controller.rb]: không dùng thì bỏ luôn controller này đi, khi nào dùng thì tạo sau

File: app/controllers/users/omniauth_callbacks_controller.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
```
Comment (dieunb) [inline app/controllers/users/omniauth_callbacks_controller.rb]: không dùng thì bỏ luôn controller này đi, khi nào dùng thì tạo sau

File: app/controllers/users/passwords_controller.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+class Users::PasswordsController < Devise::PasswordsController
```
Comment (dieunb) [inline app/controllers/users/passwords_controller.rb]: không dùng thì bỏ luôn controller này đi, khi nào dùng thì tạo sau

File: app/controllers/users/registrations_controller.rb [hunk 5]
```diff
@@ -0,0 +1,59 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
+  before_action :configure_account_update_params, only: [:update]
+  # GET /resource/sign_up
```
Comment (dieunb) [inline app/controllers/users/registrations_controller.rb]: code comment thì bỏ hết đi, dùng cái gì thì mới thêm vào

File: app/controllers/users/sessions_controller.rb [hunk 6]
```diff
@@ -0,0 +1,3 @@
+class Users::SessionsController < Devise::SessionsController
+  # before_action :configure_sign_in_params, only: [:create]
```
Comment (dieunb) [inline app/controllers/users/sessions_controller.rb]: không dùng thì bỏ luôn controller này đi, khi nào dùng thì tạo sau

File: app/controllers/users/unlocks_controller.rb [hunk 7]
```diff
@@ -0,0 +1,28 @@
+class Users::UnlocksController < Devise::UnlocksController
```
Comment (dieunb) [inline app/controllers/users/unlocks_controller.rb]: không dùng thì bỏ luôn controller này đi, khi nào dùng thì tạo sau

**Comment by dieunb** [comment]
commented

**Comment by inuyashaaa** [comment]
ready

**Comment by inuyashaaa** [comment]
ready

**Comment by dieunb** [comment]
commented

**Comment by inuyashaaa** [comment]
ready

**Pull request merged**